### ISSUE TITLE
0.7.11 breaks on pip install

### ISSUE THREAD

**Issue by michaelbrooks** [jrief/django-angular#149]
I have encountered an error with the recent change moving package.json:

```
Collecting django-angular (from -r ../requirements.txt (line 29))
  Downloading django-angular-0.7.11.tar.gz
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-s0cGie/django-angular/setup.py", line 23, in <module>
        with open('client/package.json', 'r') as fp:
    IOError: [Errno 2] No such file or directory: 'client/package.json'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

      File "<string>", line 20, in <module>

      File "/tmp/pip-build-s0cGie/django-angular/setup.py", line 23, in <module>

        with open('client/package.json', 'r') as fp:

    IOError: [Errno 2] No such file or directory: 'client/package.json'

    ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-s0cGie/django-angular
```

I can't quite figure out why it is unable to find package.json since I see it in the client folder in the source tree. Is anyone else able to install django-angular with pip?

**Comment by dasf** [comment]
Yep, I get the same error. Looking at the package from [PyPi](https://pypi.python.org/pypi/django-angular) the whole ``client`` directory seems to be missing.

**Comment by jrief** [comment]
Sorry for that.
Fixed in 0.7.12
